api/routes/assessment_routes.py

```diff
@@ -141,14 +142,13 @@ def assessment_stats_for_fund_round_id(
     stats = {}
     assessments = get_metadata_for_fund_round_id(
         fund_id=fund_id, round_id=round_id
-    )
+    )s
```

Comment:
Looks like this stray `s` is causing the tests to fail. Have you got the pre-commit hooks set-up?

Reply:
Removed and fixed, thanks!